packages/@okta/vuepress-site/docs/concepts/session/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Session management
+meta:
+  - name: description
+    content: An overview of session management in Okta. Learn about sessions in Okta and what kind of session sign out to implement.
+---
+
+# Session management with Okta
+
+An Identity Provider (IdP) is a service that stores and manages digital identities. Okta is an IdP service that stores and verifies user identities. Okta provides a federated sign-in service so that a user can access multiple applications with a single set of sign-in credentials. A session is a group of user interactions with a resource, such as an IdP service or an application (app), for a given amount of time. When you use Okta as your IdP, there are two types of sessions that you need to consider for managing the user experience&mdash;the IdP session and the application session.
+
+## IdP session
+
+The IdP session is created by Okta after the user is authenticated using their credentials and various MFA options. After authentication, the user is able to access apps within the managed identity organization (org) by using Single Sign-On (SSO), all within the scope of the IdP session. The session uses cookies to store user information to track and facilitate interactions between the user and the org. The IdP session is active until the user signs out of the org or when the session expires, based on the policies defined in the org.
```

Comment:
```suggestion
The IdP session is created by Okta after the user is authenticated by using credentials and various MFA options. After authentication, the user is able to access apps within the managed identity organization (org) by using Single Sign-On (SSO), all within the scope of the IdP session. The session uses cookies to store user information to track and facilitate interactions between the user and the org. The IdP session is active until the user signs out of the org or when the session expires, based on the policies defined in the org.
```

Reply:
Keeping the original sentence. "by entering user" is confusing to know who's credentials the user is entering.